test/property/arbitraries/canister_methods/update_method_arb.ts

```diff
@@ -136,7 +150,9 @@ export function UpdateMethodArb<
                     imports,
                     globalDeclarations,
                     sourceCode,
-                    tests
+                    tests,
+                    paramTypes,
+                    methodName: functionName
```

Comment:
Would it be worthwhile to rename methodName to indicate that it is used for typeAliases (It is used for type aliases right?) and then just use the actual method name for the method name?